content/source/docs/cloud/api/changelog.html.md

```diff
@@ -0,0 +1,58 @@
+---
+page_title: "API changelog - API Docs - Terraform Cloud and Terraform Enterprise"
+layout: "cloud"
+page_id: "api-changelog"
+---
+
+[breaking]: ./api/changelog/breaking.png "Breaking"
+[cloud]: ./api/changelog/cloud.png "Cloud"
+[enterprise]: ./api/changelog/enterprise.png "Enterprise"
+
+# API Changelog
+
+Keep track of changes to the Terraform Cloud and Terraform Enterprise APIs.
+
+### 2021-01-21
+
+* ![cloud][] A new Invoices API is available.
+* The terraform-version attribute in the Workspaces API is now deprecated. Please use terraform-requirement instead.
+* ![enterprise][] A new Module Sharing API is available for sharing Terraform modules across organizations.
+* ![breaking][] The X API will no longer return Y. To do Z, use this workaround. This is a breaking change necessary for the security of the platform, covered in our API stability policy.
+
+### 2021-05-27
+
+* Added the [delete endpoint](https://www.terraform.io/docs/cloud/api/agents.html#delete-an-agent) to the [Agents API](https://www.terraform.io/docs/cloud/api/agents.html).
+
+### 2021-04-28
+
+* Added the `refresh`, `refresh-only`, and `replace-addrs` attributes to the [Runs API](https://www.terraform.io/docs/cloud/api/runs.html).
+
+### 2021-04-16
+
+* Introduce [Controlled Remote State Access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise).
```

Comment:
@nfagerlund @omarismail much like below I only linked to the blog and added the broad strokes. Will open a discussion in Slack about this.

Reply:
👍🏼 

Let's align the verb tenses, though: 

```suggestion
* Introduced [Controlled Remote State Access](https://www.hashicorp.com/blog/announcing-controlled-remote-state-access-for-terraform-cloud-and-enterprise).
```

(There's at least one more usage to the south of here.)